projects/packages/videopress/src/client/state/actions.js

```diff
@@ -357,6 +358,41 @@ const setVideoPressSettings = videoPressSettings => {
 	return { type: SET_VIDEOPRESS_SETTINGS, videoPressSettings };
 };
 
+/**
+ * Thunk action to remove a video from the state,
+ *
+ * @param {object} settings - VideoPress settings
+ * @returns {Function}        Thunk action
+ */
+const updateVideoPressSettings = settings => async ( { dispatch } ) => {
+	if ( ! settings ) {
+		return;
+	}
+
+	const data = { force: true };
+
+	if ( settings.videoPressVideosPrivateForSite !== undefined ) {
+		data.videopress_videos_private_for_site = settings.videoPressVideosPrivateForSite;
+	}
```

Comment:
Maybe a `typeof settings.videoPressVideosPrivateForSite !== 'boolean'` would be a more strict check here.

Reply:
Addressing in a follow-up.

Reply:
you mean to `if ( settings.videoPressVideosPrivateForSite !== undefined ) {`, do you?

Reply:
Yes. Technically, it could be, say, a number.